digital_image_processing/filters/bilateral_filter.py

```diff
@@ -0,0 +1,93 @@
+"""
+Implementation of Bilateral filter
+
+Inputs:
+    img: A 2d image with values in between 0 and 1
+    varS: variance in space dimension.
+    varI: variance in Intensity.
+    N: Kernel size(Must be an odd number)
+Output:
+    img:A 2d zero padded image with values in between 0 and 1
+"""
+
+import cv2
+import numpy as np
+import math
+import sys
+
+
+def vec_gaussian(img: np.ndarray, variance: float) -> np.ndarray:
+    # For applying gaussian function for each element in matrix.
+    sigma = math.sqrt(variance)
+    cons = 1 / (sigma * math.sqrt(2 * math.pi))
+    fImg = cons * np.exp(-((img / sigma) ** 2) * 0.5)
+    return fImg
+
+
+def getSlice(img: np.ndarray, x: int, y: int, kernel_size: int) -> np.ndarray:
+    return img[
+        x - kernel_size // 2 : x + kernel_size // 2 + 1,
+        y - kernel_size // 2 : y + kernel_size // 2 + 1,
+    ]
+
+
+def getGaussKernel(kernel_size: int, spatial_variance: float) -> np.ndarray:
+    # Creates a gaussian kernel of given dimension.
+    arr = np.zeros((kernel_size, kernel_size))
+    for i in range(0, kernel_size):
+        for j in range(0, kernel_size):
+            arr[i, j] = math.sqrt(
+                abs(i - kernel_size // 2) ** 2 + abs(j - kernel_size // 2) ** 2
+            )
+    arr = vec_gaussian(arr, spatial_variance)
+    return arr
```

Comment:
Should change these method names to be `snake_case` also

Reply:
changed the style to snake_case